www/grpc/network.go

```diff
@@ -48,12 +48,16 @@ func (s *networkServer) GetNodeInfo(_ context.Context,
 }
 
 func (s *networkServer) GetNetworkInfo(_ context.Context,
-	_ *pactus.GetNetworkInfoRequest,
+	req *pactus.GetNetworkInfoRequest,
 ) (*pactus.GetNetworkInfoResponse, error) {
 	ps := s.sync.PeerSet()
 	peerInfos := make([]*pactus.PeerInfo, 0, ps.Len())
 
 	ps.IteratePeers(func(peer *peerset.Peer) bool {
+		if req.OnlyOnline && (!peer.IsConnected() || !peer.IsKnown()) {
+			return true
```

Comment:
`return true` will break the iteration (stop it).
`return false` will simply continue it.

This code will break the loop at the first occurrence. Have you tested it?

Reply:
I tested it and it worked. but since I was running the node recently, maybe all of my connections disconnected. I'll change this condition.